Hypnotic-Herons/sprint-3
# Mkmd navbar

## Link to Ticket
Closes

## Description of Proposed Changes
-added nav bar with my orders, products, sell a product, profile

## Steps to Test

```
git fetch --all
git checkout mkmd_navbar


```


1. python manage.py makemigrations website
1. python manage.py migrate
1. python manage.py runserver
1. go to url  http://127.0.0.1:8000
1. make sure you are logged out
1. you should NOT see nav bar if you are logged out
1. register as a user 
1. log in with your username and password
1. you should see the nav bar once you are logged in with your user name at the right of the buttons
1. click on sell a product button and add some products to sell
1. click on products and you should see a list of products
1. orders button and profile button will not work until we complete those tickets


## Impacted Areas in Application
- if user is authenticated, you should see navbar


## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
